test/integration/consul-container/test/ratelimit/ratelimit_test.go

```diff
@@ -68,7 +68,7 @@ func TestServerRequestRateLimit(t *testing.T) {
 	testCases := []testCase{
 		// HTTP & net/RPC
 		{
-			description: "HTTP & net/RPC / Mode: disabled - errors: no / exceeded logs: no / metrics: no",
+			description: "HTTP & net-RPC | Mode: disabled - errors: no | exceeded logs: no | metrics: no",
```

Comment:
I replaced `/` with `|` because vscode uses `/` for the test name delimiter in it's test runner and it wreaked all sorts of havoc. Open to other suggestions.